charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 1.1.0
+version: 1.1.1
```

Comment:
This is adding a feature to the chart so I think the version should be 1.2.0

Reply:
Good call. Fix pushed.